src/utils/daemon.js

```diff
@@ -61,3 +66,48 @@ export default async function createDaemon (opts) {
 
   return ipfsd
 }
+
+async function cleanup (opts) {
+  const option = dialog.showMessageBox({
+    type: 'warning',
+    title: 'IPFS Desktop',
+    message: `We coudln't start because IPFS didn't shutdown correctly last time. We can try to
```

Comment:
There is a typo in coudln't and, let's refer to IPFS Desktop rather than `We`

```suggestion
    message: `IPFS Desktop found an api file in ${IPFS_HOME}, but was unabled to connect to the address ${apiAddr}`. Would you like to try running `ipfs repo fsck` to remove the file and try again?
```